core/src/models/Profile.ts

```diff
@@ -109,7 +109,7 @@ export class Profile extends LoggedModel<Profile> {
 
   async addOrUpdateProperties(
     properties: {
-      [key: string]: Array<string | number | boolean | Date>;
+      [key: string]: Array<string | number | boolean | Date> | any;
```

Comment:
That's a shame... it's been really helpful to enforce the Array type.   What required this change?

Reply:
Re-checked and it wasn't really necessary. Moved back to using arrays!